### ISSUE TITLE
A sample issue

### ISSUE THREAD

**Issue by rpottsoh** [TMSI-LLC/software-discussions#1]
This is just a test issue, to be an example for others.......

**Comment by rpottsoh** [comment]
@thetribefan welcome to github.  This is just sample issue.  Because I have mentioned you by name you should be receiving this post as an email.  You can also see this post by clicking the view in github link at the bottom of the email.  You can also add a comment to this issue by either posting it on the website or by replying to this email.  One suggestion though, when replying by email, do not include the whole email you received as quoted text.  Either include no quoted text, or only relevant portions.  All the comments in the issue are available online.

**Comment by thetribefan** [comment]
This seems like Spiceworks to me.

**Comment by rpottsoh** [comment]
It might have that `feel`, but this is a version control system.

**Comment by rpottsoh** [comment]
You can do most of the same sorts of things that Spiceworks does with the added benefit that this has the potential to replace Code Co-Op. :+1:

**Comment by thetribefan** [comment]
Right.  I understood that from the _git_ go.  It will just take some time for me to acclimate to its use.  I saw your tutorial.  In due time...I will look at it.

**Comment by rpottsoh** [comment]
no rush.  What I haven't shared with you is how to deal (work) with code.
I have started a pull request from my forked copy
tmsi-llc/software-discussions.  You are able to review the PR and can make
suggestions for changes or even suggest the changes (via the web
interface).  A dialog can ensue between the person issuing the PR and those
responsible for deciding if the content of the PR is OK or not.  If
accepted the commit(s) that make up the PR are pulled from the issuers copy
of the repo and placed into the same place in the main repo.  Brief, in a
nut shell, explanation.

Ryan
